### ISSUE TITLE
`RunOptions` `dataclass` uses mutable default value

### ISSUE THREAD

**Issue by jakirkham** [maccallumlab/meld#160]
Recently ran into this error ( https://github.com/conda-forge/meld-feedstock/pull/24#issuecomment-1847698955 ). Appears other users have seen it as well ( https://github.com/maccallumlab/meld/issues/133#issuecomment-1552970732 )

```python
ValueError: mutable default <class 'openmm.unit.quantity.Quantity'> for field pressure is not allowed: use default_factory
```

Looks like it traces back to this line

https://github.com/maccallumlab/meld/blob/32000a0edaeb1f291d9d301cc2cadb9304c90575/meld/system/options.py#L26

Based on the Python docs, think we want to use [a pattern like this]( https://docs.python.org/3/library/dataclasses.html#dataclasses.field ). Could also wrap it in a `lambda` or use `functools.partial` to bind arguments to `operator.mul`. Something that gives us a no option function to use with `default_factory` for this field

**Comment by jlmaccal** [comment]
This is already fixed on the master branch.

**Comment by jakirkham** [comment]
Ah I see. Thanks Justin! 🙏 

https://github.com/maccallumlab/meld/blob/8125432029f87fe78167e6f948071cd0785f9860/meld/system/options.py#L25

Is there a rough idea of when the next release might be?

**Comment by jlmaccal** [comment]
Not really. I suspect we have some work to do to with the packaging scripts. We also need to move to OpenMM 8. I don't think it's too difficult, but I have very little time to work on this right now.
